src/core/lib/debug/stats_data.cc

```diff
@@ -281,7 +276,6 @@ const char* grpc_stats_histogram_doc[GRPC_STATS_HISTOGRAM_COUNT] = {
     "Number of streams whose payload was written per TCP write",
     "Number of streams terminated per TCP write",
     "Number of flow control updates written per TCP write",
-    // NOLINTNEXTLINE(bugprone-suspicious-missing-comma)
```

Comment:
Was this removal intentional?  Seems unrelated to the rest of the PR.

Reply:
looks like someone made a manual edit to an auto-generated file?

Reply:
@veblush can we just have clang-tidy not look at this file?

Reply:
Two things we can do. i) modifying `tools/codegen/core/gen_stats_data.py` to have this comment (I didn't know this is auto-generated, either) or ii) modifying tools/dockerfile/grpc_clang_tidy/clang_tidy_all_the_things.sh to exclude this file. I prefer 1 since it'd be valuable to run clang-tidy even on generated files since the generator could be buggy. But i'm not strongly opposing to the second option, either.

Reply:
I think it happens after code generation in clang-format, which is why I'm thinking we just skip it.

Reply:
removed them from clang-tidy for now